### PR TITLE
docs(UI.java, CalculatorTest.java, Library): changes text type, style and size, added delta to assertEquals, added jars file

### PR DESCRIPTION
**Pull request by junwei991021**
-changes text type, style and size in UI.java; added delta for assertEquals CalculatorTest.java; added external jars file

### CODE REVIEW

**Comment by junwei991021** [comment]
Added jars file, changes text size, type and style, fix the test code bug

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pH-7) [inline .project:1]: This shouldn't be added here :)

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@
 # Mac OS System
 .DS_Store*
 ._*
+/bin/
```
Comment (pH-7) [inline .gitignore:9]: Can you please not modify .gitignore in this PR. Just keep focusing on the original purpuse of your PR 😉  Thanks!

File: .classpath [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pH-7) [inline .classpath:1]: Can you please remove this one from this pull requestr? Appreciate 😄

**Comment by pH-7** [comment]
Thanks very much for your great contribution @junwei991021  Please see above some suggestions. Please have a look and let me know. 


Thanks!

**Comment by pH-7** [comment]
@junwei991021 You will also need to fix a minor conflict in `src/simplejavacalculatorTest/CalculatorTest.java`

Update the branch with master, then you will see the conflict to fix 🙂 Thanks

File: src/simplejavacalculatorTest/CalculatorTest.java [hunk 4]
```diff
@@ -14,69 +14,69 @@
...
 		Calculator calculator = new Calculator();
 		calculator.calculateBi(Calculator.BiOperatorModes.add, 3.0);
-		Assertions.assertEquals(5.5, calculator.calculateBi(Calculator.BiOperatorModes.normal, 2.5));
+		Assertions.assertEquals(5.5, calculator.calculateBi(Calculator.BiOperatorModes.normal, 2.5),0.0);
 	}
 	
 	@Test
 	void calculateBiMinusTest() {
 		Calculator calculator = new Calculator();
 		calculator.calculateBi(Calculator.BiOperatorModes.minus, 3.1415);
```
Comment (Amanda1232) [inline src/simplejavacalculatorTest/CalculatorTest.java:30]: Our static analysis engine detects that your pull request includes a change that uses the rough value of Math.PI. According to the [best practice](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value), it's recommended to use the predefined library constant for code clarity and better precision.

**Comment by pH-7** [comment]
Closing this one as we didn't hear from the author. Anyways, you are welcome to open another PR later on with correct changes @junwei991021

**Pull request closed without merging**